Source/gui/SNTNotificationManager.m

```diff
@@ -444,6 +444,12 @@ - (void)didRegisterForAPNS:(NSString *)deviceToken {
 }
 
 - (void)APNSTokenChanged {
+  // Only message the sync service if a sync server is configured and APNS is enabled, otherwise the
+  // service will needlessly spin up.
+  // TODO: To realize changes to EnableAPNS, both the gui and sync service need to be restarted. Add
```

Comment:
I think this needs to be resolved before 2025.4 is released

Reply:
I made https://github.com/northpolesec/santa/issues/356